packer/plugin/server.go

```diff
@@ -38,12 +38,14 @@ const MagicCookieValue = "d602bf8f470bc67ca7faa0386276bbdd4330efaf76d1a219cb4d69
 // know how to speak it.
 const APIVersion = "5"
```

Comment:
Oh right, yes !

Reply:
```suggestion
const APIVersion = "6"
```

Reply:
Ah wait, while trying to explain why this was needed in the commit I realised that this won't break the API. This adds a new binary type that Packer can start. But old plugins won't need to re-compile because of _this_ PR.

Do you agree ?

Reply:
new plugins won't know if the version of packer is compatible with _them_ ... does this matter?

Reply:
🤔 I don't think so: New plugins ( packer-plugin-* ) can only be started by a Packer that knows how to start them.

With all that said, we will need to do that bump anyway if/when we introduce the new datasource plugin type.

Reply:
Okay, I think this is good to merge, and we _will_ bump that version a bit later on 🙂 . For now I _think_ this is still okay. :D

I also think that when we do the next bump it would be nice to introduce a APIVersionMinor which will be set to zero and not used, to allow for backward compatible api adds.